app/assets/templates/status-message-location_tpl.jst.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if location}}
   <div class='near-from'>
-    {{ t "publisher.near_from" location=location}}
+    <a href="/posts/{{id}}">{{ t "publisher.near_from" location=location}}</a>
```

Comment:
I didn't realize that you're linking to the SPV, I thought you'd already pop-in the map when clicking on the location.